scripts/release/rpm/Dockerfile.mariner

```
@@ -1,6 +1,6 @@
-ARG tag=1.0
+ARG tag=2.0.20211216-amd64
```

I think we should use the 2.0 tag instead pinning to a particular version.
/cc @jslobodzian @nicogbg
```suggestion
ARG tag=2.0
```